src/templates/exhibition.html

```diff
@@ -3,10 +3,10 @@
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
 <title><% title %></title>
-    <link rel="stylesheet" href="<% gallery_root %>css/mediaboxAdvWhite.css" type="text/css" media="screen" />
+    <link rel="stylesheet" href="<% gallery_root %>images/mediaboxAdv-Light.css" type="text/css" media="screen" />
```

Comment:
Oops?!
Lets rather move the whole mediabox to a separate directory than mixing images and CSS code for the whole project. If we do that, we maybe can even remove it completely and add a git submodule (or something like that) to directly reference to the original project instead of keeping a copy here.